src/spaces/grid.jl

```diff
@@ -41,11 +42,14 @@ cartesian indices have Euclidean distance `≤ r` from the cartesian index of th
 position.
 
 An example using `GridSpace` is the [Forest fire model](@ref).
+
+TODO pathfinder
 """
 function GridSpace(
     d::NTuple{D,Int};
     periodic::Bool = true,
     metric::Symbol = :chebyshev,
+    pathfinder::Union{NamedTuple,Nothing}=nothing,
```

Comment:
Hm... This approach of having secondary keyword arguments passed to keyword arguments is a bit clunky... But I guess at the moment is also the best we can do. I thought about it but couldn't really get a better alternative. We should just be sure to explicitly do this in teh docs:
```julia
pathfinder = (:admissability = ..., :walkable = ...)
space = GridSpace(d; pathfinder)
```

Reply:
EDIT: I suggest in the main comment of the review to define `Pathfinder` which also contains all documentation. That's how we make this simpler.

Reply:
So in the docs of the `GridSpace` we say: "`pathfinder`: An instance of `[PathFinder](@ref)` that enables path-finding capabilities based on the A* algorithm".